vysledek/style.css

```diff
@@ -0,0 +1,327 @@
+    font-size: 16px;
+    line-height: 1.6;
+}
+
+.leading {
+    display: flex;
+    flex-basis: 100%;
+    flex-direction: column;
+    font-family: 'Raleway', sans-serif;
+    font-weight: 600;
+    justify-content: space-around;
+    min-width: 440px;
+    background-color: white;
+
+}
+
+/*                  HEADER                              */
+header {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    background-image: url("/images/island-hlavicka.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    color: white;
+    text-align: center;
+    font-weight: 700;
+    padding-left: 25px;
+    padding-right: 25px;
+    padding-bottom: 25px;
+    min-width: 440px;
+    font-family: 'Raleway', sans-serif;
+}
+
+
+header h1 {
+    font-size: 3.5rem;
+}
+
+.signature {
+    color: lightgray;
+    position: relative;
+}
+
+.signature::before {
+    position:absolute;
+    content: '';
+    top:-20px;
+    height: 3px;
+    margin: 0 auto;
+    right: 0;
+    left: 0;
+    width: 17%;
+    background: lightgray;
+}
+/*--------------                                    */
+
+
+
+/*                  NAVIGACE                        */
+nav a {
+    text-decoration: none;
+    text-transform: uppercase;
+    color: lightgray;
+}
+
+ul {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    justify-content:center;
+    text-align: center;
+    padding: 50px;
+}
+
+li {
+    list-style-type: none;
+    text-decoration: none;
+    border-bottom: solid 1px lightgray;
+}
+
+/*                  TEXT SECTION                  */
+h2 {
+    padding-left: 20px;
+}
+
+.text {
+    font-size: 1rem;
+    padding-top: 20px;
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-bottom: 30px;
+    font-weight: 600;
+}
+
+.underline {
+    text-decoration: none;
+    position: relative;
+}
+
+.underline::after {
+        position: absolute;
+        content: '';
+        height: 5px;
+        bottom: -4px;
+        margin: 0 auto;
+        left: 0;
+        width: 17%;
+        background: #629edb;
+}
+
+.autor {
+    color: gray;
+    font-size: 0.9rem;
+    text-align: center;
+    margin: 0px;
+    padding-top: 3px;
+}
+
+.autor a {
+    color: gray;
+    font-size: 0.9rem;
+}
+
+/*                  GALLERY                     */
+.gallery {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    padding-top: 40px;
+}
+
+.label {
+    color: gray;
+    font-size: 0.8rem;
+    font-weight: 800;
+    text-align: center;
+    margin: 0;
+    padding: 0;
+}
+
+.description a {
+    color: gray;
+}
+
+.foto {
+    padding-bottom: 20px;
+}
+
+/*                  FOOTER                      */
+footer {
+    background-color: #f5f5f5;
+    color: gray;
+    padding: 20px;
+    font-size: 0.75rem;
+    font-weight: 700;
+
+}
+footer a {
+    color: gray;
+}
+
+
+
+/*TABLET*/
+@media (min-width: 500px) {
+    ul {
+        display: flex;
```

Comment:
Tady už flex speciikovat nemusíš, máme ho už definovaný v mobile-first stylech a v media queries chceme jen "přepisovat" to, co má být jinak